eng/Subsets.props

```diff
@@ -101,7 +101,7 @@
 
   <ItemGroup>
     <!-- CoreClr -->
-    <SubsetName Include="Clr" Description="The CoreCLR runtime, LinuxDac, CoreLib (+ native), tools and packages." />
+    <SubsetName Include="Clr" Description="The full CoreCLR runtime: $(DefaultCoreClrSubsets)" />
```

Comment:
@MichalStrehovsky @jkotas Does it make sense that `build.cmd clr` builds nativeaotlibs but not nativeaotruntime?

Reply:
Clr includes clr.native and that's all native code under src/coreclr including clr.nativeaotruntime, clr.iltools, etc.

I just realized I put `clr.spmi` into the `DefaultCoreClrSubsets` but that one is also redundant with `clr.native`. We could remove it; it's a NOP.